modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/beanValidation.mustache

```diff
@@ -0,0 +1,53 @@
+{{#required}}
```

Comment:
thx for the hint, the templates are now reused in beanValidationParams.mustache.